### DISCUSSION TITLE
eslint error: Unable to resolve path to module 'swiper/modules'

### DISCUSSION THREAD

**Discussion by ckanitz** [nolimits4web/swiper#6849] [Q&A]
with version 10 the module imports moved from `swiper` to `swiper/modules`
updates my scripts and build with webpack works again

eslint however can't resolve the path to `swiper/modules` resulting in an error.

for now I resolved it with `// eslint-disable-next-line import/no-unresolved` but a solution to "hint" eslint correctly would be nice.

any ideas?

**Comment by adropofilm** [comment]
@ckanitz did you ever figure this out? Running into the same issue.

**Reply by Kate-Shepel** [reply]
did you ever figure this out? Running into the same issue :(

**Reply by ckanitz** [reply]
Nope, it's still an issue

**Comment by matdave** [comment]
Glad I'm not the only one. So basically, it needs either a main or index file in the module's directory instead of nothing.
